ci-operator/step-registry/strimzi/run-tests/strimzi-run-tests-commands.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "Running the tests"
+./run_tests.sh
+
+echo "Copy logs and xunit to artifacts dir"
+./copy_logs.sh "${ARTIFACT_DIR}"
```

Comment:
The test results for this scenario are reported to Testgrid. In order for us to post the tests from the xmls we need to prefix the xmls file with junit so that they are auto-discovered by Testgrid.

Here's an [example where we needed to do this for our MTR scenario ](https://github.com/openshift/release/pull/37911) 

It looks like we are generating 6 xmls from this test [here](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/37253/rehearse-37253-periodic-ci-amq-streams-qe-strimzi-kafka-operator-main-amq-streams-ocp4.13-lp-interop-amq-streams-2-3/1642917620778799104/artifacts/amq-streams-2-3/strimzi-run-tests/artifacts/xunit/)

So I believe we can just do something like
```
mv ${ARTIFACT_DIR}/TEST-io.strimzi.systemtest.connect.ConnectBuilderIsolatedST.xml ${ARTIFACT_DIR}/junit_TEST-io.strimzi.systemtest.connect.ConnectBuilderIsolatedST.xml 
mv ${ARTIFACT_DIR}/TEST-io.strimzi.systemtest.connect.ConnectIsolatedST.xml ${ARTIFACT_DIR}/junit_TEST-io.strimzi.systemtest.connect.ConnectIsolatedST.xml
mv ${ARTIFACT_DIR}/TEST-io.strimzi.systemtest.cruisecontrol.CruiseControlST.xml ${ARTIFACT_DIR}/junit_TEST-io.strimzi.systemtest.cruisecontrol.CruiseControlST.xml
mv ${ARTIFACT_DIR}/TEST-io.strimzi.systemtest.kafka.listeners.ListenersST.xml ${ARTIFACT_DIR}/junit_TEST-io.strimzi.systemtest.kafka.listeners.ListenersST.xml
mv ${ARTIFACT_DIR}/TEST-io.strimzi.systemtest.metrics.MetricsIsolatedST.xml ${ARTIFACT_DIR}/junit_TEST-io.strimzi.systemtest.metrics.MetricsIsolatedST.xml
mv ${ARTIFACT_DIR}/failsafe-summary.xml ${ARTIFACT_DIR}/junit_failsafe-summary.xml
```

Reply:
Name of th test class can be changed so it is not a great idea to hardcode the names here. We can copy files with prefix only to avoid unnecessary files copied as well, but copy_logs.sh also copy logs gathered by test-suite for debug in case of failure so we need to copy this as well. I opened PR to enhance our script, will share it on slack.

Reply:
Just saw the [enhancement](https://gitlab.cee.redhat.com/strimzi/amq-streams-interop/-/merge_requests/8) was merged this is a great solution!